### PR TITLE
Corrected BaseCommand.check() signature in docs.

### PR DESCRIPTION
**Pull request by evananyonga**

### CODE REVIEW

File: docs/howto/custom-management-commands.txt [hunk 1]
```diff
@@ -305,16 +305,16 @@ the :meth:`~BaseCommand.handle` method must be implemented.
...
 
     Uses the system check framework to inspect the entire Django project for
     potential problems. Serious problems are raised as a :exc:`CommandError`;
     warnings are output to ``stderr``; minor notifications are output to
     ``stdout``.
 
     If ``app_configs`` and ``tags`` are both ``None``, all system checks are
-    performed. ``tags`` can be a list of check tags, like ``compatibility`` or
-    ``models``.
+    performed except deployment checks which default to False. ``tags`` can be a list of check tags, like ``compatibility`` or
```
Comment (felixxm) [inline docs/howto/custom-management-commands.txt:316]:
```suggestion
    performed except deployment and database related checks. ``tags`` can be a
    list of check tags, like ``compatibility`` or ``models``.
```

**Review by felixxm — COMMENTED** [review]
@evananyonga Thanks :+1: I left suggestions.

**Comment by felixxm** [comment]
@evananyonga Do you have time to keep working on this?

**Comment by evananyonga** [comment]
I have some time again. I am working on your recommendations

File: docs/howto/custom-management-commands.txt [hunk 2]
```diff
@@ -305,16 +305,16 @@ the :meth:`~BaseCommand.handle` method must be implemented.
...
+    ``models``. Database checks will be skipped with ``databases`` set to the default None.
```
[the remark this answers is not in the document]
Comment (evananyonga) [reply docs/howto/custom-management-commands.txt:317]: Hey @felixxm should I change the entire instruction for the new suggestion or should it just be an addition?

[the remark this answers is not in the document]
Comment (felixxm) [reply docs/howto/custom-management-commands.txt:317]: I'd [add](https://github.com/django/django/pull/17764/files#r1461448061) _"and database related checks"_ to the existing description, and a [new sentence](https://github.com/django/django/pull/17764/files#r1461453497) about `include_deployment_checks` and `databases` below.

[the remark this answers is not in the document]
Comment (evananyonga) [reply docs/howto/custom-management-commands.txt]: Completed

**Comment by felixxm** [comment]
@evananyonga Thanks for updates :+1:

**Pull request merged**